internal/webpreview/overlay_chat_test.go

```
@@ -15,7 +15,7 @@ func TestOverlayModulesJS(t *testing.T) {
 	if err != nil {
 		t.Skip("node not installed; skipping overlay module tests")
 	}
-	cmd := exec.Command(node, "--test", "chat_test.mjs", "settings_test.mjs", "sourcecontrol_test.mjs", "boxpos_test.mjs")
+	cmd := exec.Command(node, "--test", "chat_test.mjs", "markdown_test.mjs", "settings_test.mjs", "sourcecontrol_test.mjs", "boxpos_test.mjs", "launcher_test.mjs")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)